packages/realm_dart/ffigen.yaml

```diff
@@ -27,6 +27,7 @@ comments:
 compiler-opts:
   - '-DRLM_NO_ANON_UNIONS'
   - '-DFFI_GEN'
+  - '-DREALM_APP_SERVICES'
```

Comment:
this should be on, by default, shouldn't it? Why do we have to define it here?

Reply:
It is on by default in cmake. This is the ffigen config.